tests/unit/test_dbt_containerizer.py

```diff
@@ -0,0 +1,32 @@
+import os
+from pathlib import Path
+from palm.containerizer import DbtContainerizer
+
+class MockContext:
+    def __init__(self, **kwargs):
+        self.__dict__.update(kwargs)
+
+DbtContainerizer.__abstractmethods__ = set()
+
+def test_run(tmp_path, environment):
```

Comment:
I don't think the `environment` fixture is defined?

Reply:
resolved